ui/util/lbryURI.js

```diff
@@ -103,17 +103,18 @@ export function parseURI(url: string, requireProto: boolean = false): LbryUrlObj
   }
 
   // Validate and process modifier
-  const [primaryClaimId, primaryClaimSequence, primaryBidPosition] = parseURIModifier(
+  const [primaryClaimId, primaryClaimSequence, primaryBidPosition, primaryPathHash] = parseURIModifier(
```

Comment:
The `primaryPathHash` additional should probably be repeated in the duplicate `web/src` version too, until this was intentionally left out and unneeded for web.

Reply:
Forgot to mention that I squashed a bit and rebased.